examples/rollbar-spring-webmvc/.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea
```

Comment:
I think this file is not needed as all the ignores are already at the `.gitignore` file located at root level.

Reply:
done